src/sentry/api/endpoints/organization_member/utils.py

```diff
@@ -33,3 +36,19 @@
 
 class MemberConflictValidationError(serializers.ValidationError):
     pass
+
+
+class RelaxedMemberPermission(OrganizationPermission):
+    scope_map = {
+        "GET": ["member:read", "member:write", "member:admin"],
+        "POST": ["member:write", "member:admin"],
+        "PUT": ["member:invite", "member:write", "member:admin"],
+        # DELETE checks for role comparison as you can either remove a member
+        # with a lower access role, or yourself, without having the req. scope
+        "DELETE": ["member:read", "member:write", "member:admin"],
```

Comment:
for later: we should probably allow deleting an invite/request if the requesting user was the inviter, regardless of scopes